src/lib/flocking/ugens/envelopes.js

```diff
@@ -0,0 +1,1101 @@
+/*
```

Comment:
We should source this material from npm or bower rather than committing it wholesale